AGENTS.md

```
@@ -0,0 +1,352 @@
+
+### 코드베이스 개요
+
+- **루트 파일**: `build.gradle.kts`, `settings.gradle.kts`, `gradle.properties`, `gradlew`, `LICENSE`, `README.md`, `renovate.json`
+- **아키텍처**: Feature → Domain → Data, Core는 모든 레이어에서 공유
+- **언어**: Kotlin, UI는 Jetpack Compose 사용
+- **빌드 시스템**: `build-logic/`의 컨벤션 플러그인, 커스텀 DSL(`kuring()`, `kuringPrimitive()`) 사용
+- **모듈 수**: 4개 주요 레이어로 구성된 40개 이상의 모듈
+
+### 기술 스택
+
+**핵심 프레임워크**
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)